harvard-lts/jstor_aspace_oai_harvester
# new mongo collection support

**use new mongo collectioons**
* * *

**JIRA Ticket**: n/a

# What does this Pull Request do?
harvester now uses 2 collections: jstor_harvested_summary (summary for each harvest) and jstor_harvested_records (record-level harvest logging)

# How should this be tested?
run the itest or send a message via http post. you should see records in the collections described above and logging to reflect the mongo connections

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
